Create ui-c8y-1022-11-0-improved-visual-cues-for-expiring-trusted-certificates-and-renewal-option-9579.md (#3594)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1022-11-0-improved-visual-cues-for-expiring-trusted-certificates-and-renewal-option-9579.md

---
date: ""

> title: Improved visual cues for expiring trusted certificates and renewal option

Reviewer: ```suggestion
title: Improved highlighting of expiring trusted certificates and added renewal option
```

product_area: Device management & connectivity
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-62992
version: 1022.11.0
---

> This change improves the highlighting of the expiring trusted certificates in Device Management > Management > Trusted certificates. It also adds an option to renew a CA certificate if it's reaching its expiration date.

Reviewer 2: ```suggestion
To make it easier for users to identify expiring trusted certificates, visual cues have been improved to highlight certificates that are close to expiration in the Device Management > Management > Trusted certificates view. Additionally, a new option has been introduced allowing users to directly renew a CA certificate from the UI if it is approaching its expiration date. These enhancements will help users to more effectively monitor and manage the lifecycle of their trusted certificates.
```

> To make it easier for users to identify expiring trusted certificates, visual cues have been improved to better highlight certificates that are close to expiration in the Device Management > Management > Trusted certificates view. Additionally, a new option has been introduced allowing users to directly renew a CA certificate from the UI if it is approaching its expiration date. These enhancements will help users to more effectively monitor and manage the lifecycle of their trusted certificates.

Reviewer: ```suggestion
In the **Trusted certificates** page in the Device Management application, visual cues have been improved to highlight expiring certificates. Additionally, a new option has been introduced that allows users to directly renew a CA certificate from the UI if it is approaching its expiration date. These enhancements help users to monitor and manage the lifecycle of their trusted certificates more effectively.
```

> To make it easier for users to identify expiring trusted certificates, visual cues have been improved to better highlight certificates that are close to expiration in the Device Management > Management > Trusted certificates view. Additionally, a new option has been introduced allowing users to directly renew a CA certificate from the UI if it is approaching its expiration date. These enhancements will help users to more effectively monitor and manage the lifecycle of their trusted certificates.

Reviewer 2: "certificates that are close to expiration" has already been replaced with "expiring certificates"
